From 6c7424d35d1e55982f4b9a86b8a4c6f13b91efec Mon Sep 17 00:00:00 2001
From: kalyan kondapally <kalyan.kondapally@intel.com>
Date: Thu, 18 Aug 2016 16:05:58 +0800
Subject: [PATCH 3/7] HACK: Fix boot issues on SKL NUC.

Lets not bail out if SELinux domain is not defined.
---
 init/service.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/init/service.cpp b/init/service.cpp
index 3149f8e..3cd3cd2 100644
--- a/init/service.cpp
+++ b/init/service.cpp
@@ -373,9 +373,6 @@ bool Service::Start() {
         }
         if (rc == 0 && scon == mycon) {
             ERROR("Service %s does not have a SELinux domain defined.\n", name_.c_str());
-            free(mycon);
-            free(fcon);
-            return false;
         }
         free(mycon);
         free(fcon);
--
2.7.4
